src/app/routes/liveRadio/getInitialData/index.js

```diff
@@ -19,14 +22,62 @@ const getPageIdentifier = path([
 const getHeading = path(['content', 'blocks', 0, 'text']);
 const getBodySummary = path(['content', 'blocks', 1, 'text']);
 
-export default async ({ path: pathname, pageType }) => {
+export const hasRadioSchedule = async (service, pathname) => {
+  const config = await getConfig(service);
+
+  // onLiveRadioPage is enabled on Persian to render the schedule for bbc_dari_radio
+  // however bbc_persian_radio should not show the schedule
+  if (service === 'persian' && pathname.includes('bbc_persian_radio')) {
+    return false;
+  }
+
+  const serviceHasRadioSchedule = pathOr(
+    false,
+    ['radioSchedule', 'hasRadioSchedule'],
+    config,
+  );
+
+  const radioScheduleOnLiveRadioPage = pathOr(
+    false,
+    ['radioSchedule', 'onLiveRadioPage'],
+    config,
+  );
+
+  return serviceHasRadioSchedule && radioScheduleOnLiveRadioPage;
+};
+
+const radioServices = {
+  indonesia: 'indonesian',
+  persian: 'dari',
+  afaanoromoo: 'oromo',
+  bengali: 'bangla',
+};
+
+const getRadioService = service => {
+  return radioServices[service];
+};
```

Comment:
Could we use the value of the masterbrand (included in `path`) instead of mapping service? (This is fine for now, perhaps something we could address in a future PR)